src/tests/ftest/util/verify_perms.py

```diff
@@ -260,12 +260,10 @@ def _real_w(entry_type, path):
     '''
     if entry_type == 'file':
         try:
-            # Write a newline to reduce likelihood of corrupting an executable file
+            # Always write a line that allows the file to be executable
```

Comment:
It doesn't really matter if we corrupt the file, since this is only for test purposes anyway